libraries/AP_HAL_ESP32/Util.h

```diff
@@ -54,9 +58,10 @@ class ESP32::Util : public AP_HAL::Util
     bool get_system_id(char buf[50]) override;
     bool get_system_id_unformatted(uint8_t buf[], uint8_t &len) override;
 
-#ifdef HAL_PWM_ALARM
-    bool toneAlarm_init() override;
+#if AP_NOTIFY_TONEALARM_ENABLED
+    bool toneAlarm_init(uint8_t types) override;
     void toneAlarm_set_buzzer_tone(float frequency, float volume, uint32_t duration_ms) override;
+    static uint8_t _toneAlarm_types;
```

Comment:
I think this variable should be a PRIVATE variable and the STATIC designation is unnecessary.